stable/pipelines/README.md

```diff
@@ -81,6 +80,21 @@ kubectl create ns pipelines
 helm upgrade --install pipelines --namespace pipelines center/jfrog/pipelines -f pipelines/values-ingress.yaml -f pipelines/values-ingress-passwords.yaml
 ```
 
+### Special Upgrade Notes
+
+While upgrading from Pipelines 1.x to 2.x and above charts, Due to breaking changes in rabbitmq subchart (6.x to 7.x chart version when `rabbitmq.enabled=true`) and postgresql subchart(8.x to 9.x chart version when `postgresql.enabled=true`) please run below manual commands (downtime is required)
```

Comment:
`Pipelines 1.x to 2.x` might be interpreted as app versions. Can you rephrase this to make it clear that we're talking about chart versions? Maybe something like `While upgrading from Pipelines chart version 1.x to 2.x or above...`

Reply:
Sure

Reply:
Done